packages/office-ui-fabric-react/src/components/Button/ButtonDefault.tsx

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import {
+  css,
+  getNativeProps,
+  buttonProperties
+} from '../../Utilities';
+
+import { ButtonBase } from './ButtonBase';
+import './sass/ButtonBase.scss';
+import './sass/ButtonDefault.scss';
+
+export class DefaultButton extends ButtonBase {
+  protected getRootClassName() { return 'ms-Button--default'; }
+
+  public render() {
```

Comment:
which is kinda troubling as well as buttonBase currently has all children dropped into Button-label.